projects/exui-common-lib/src/lib/components/loading-spinner/loading-spinner.component.ts

```diff
@@ -8,16 +8,16 @@ import { Component, Input, OnDestroy, OnInit, Renderer2, ViewEncapsulation } fro
 })
 
 export class LoadingSpinnerComponent implements OnInit, OnDestroy {
-  @Input() zIndex: number = 1000;
-  @Input() loadingText: string = 'Page can take up to 30 seconds to load';
+  @Input() public zIndex: number = 1000;
+  @Input() public loadingText: string = 'Page can take up to 30 seconds to load';
 
-  constructor(private renderer: Renderer2) {}
+  constructor(private readonly renderer: Renderer2) {}
 
-  ngOnInit() {
+  public ngOnInit() {
```

Comment:
If we're going to introduce `public` at this stage, we should also give it a `: void` return type.

Reply:
done